source/components/Autocomplete.js

```diff
@@ -175,10 +175,10 @@ class AutocompleteBase extends Component<AutocompleteProps, State> {
     this._setInputValue(value);
     if (hasMultipleValues) {
       this.open();
-      setTimeout(() => {
-        this.updateSelectedOptions(event, multipleValues);
-      }, 0);
     }
+    setTimeout(() => {
+      this.updateSelectedOptions(event, multipleValues);
+    }, 0);
```

Comment:
@aleksandardjordjeviciohk can you explain why this timeout is necessary?

Reply:
This timeout was added to prevent issues with opening up/hiding suggestions dropdown as the issues occured when you start typing fast